src/engine/ruleset/schemas/wazuh-logpar-types.json

```diff
@@ -1119,6 +1106,7 @@
         "wazuh.origin": "keyword",
         "wazuh.registered_ip": "keyword",
         "wazuh.source": "keyword",
+        "wazuh.dns.question.original": "text",
```

Comment:
Change field `wazuh.dns.question.original` to `suricata.dns.question`
```suggestion
        "suricata.dns.question.original": "text",
```

Reply:
this information already exist in dns.question.name